src/AdminConsole/Components/Pages/App/Settings/SettingsComponents/AuthenticationConfiguration.razor

```diff
@@ -0,0 +1,117 @@
+                                </ConfirmEditForm>
+                            }
+                        </td>
+                    </tr>
+                }
+            }
+        </Table>
+    </Panel>
+}
+
+@code {
+    private string AppId => CurrentContext.AppId!;
+
+    [SupplyParameterFromForm(FormName = nameof(ConfirmDeleteAuthenticationConfigurationModel))]
+    public ConfirmSelectedFormModel ConfirmDeleteAuthenticationConfigurationModel { get; set; } = new();
+
+    private readonly IReadOnlyCollection<string> _requiredAuthenticationConfigurationNames = new[] { SignInPurpose.SignInName, SignInPurpose.StepUpName };
+
+    private IReadOnlyCollection<AuthenticationConfigurationEntry> Configurations { get; set; } = Array.Empty<AuthenticationConfigurationEntry>();
+
+    public record AuthenticationConfigurationEntry(
+        string Purpose, 
+        string UserVerification, 
+        string TimeToLive,
+        DateTimeOffset? LastUsedOn, 
+        bool CanDelete);
+
+    private AuthenticationConfigurationEntry GetEntry(Common.Models.Apps.AuthenticationConfiguration configuration) =>
+        new(configuration.Purpose,
+            configuration.UserVerificationRequirement.Humanize(LetterCasing.Title),
+            TimeSpan.FromSeconds(configuration.TimeToLive).Humanize(),
+            configuration.LastUsedOn,
+            _requiredAuthenticationConfigurationNames.All(x => !string.Equals(x, configuration.Purpose, StringComparison.OrdinalIgnoreCase)));
+
+    protected override async Task OnInitializedAsync()
+    {
+        if (CurrentContext.IsPendingDelete) return;
+
+        if (HttpContextAccessor.IsRazorPages() && HttpContextAccessor.HttpContext!.Request.HasFormContentType)
+        {
+            var request = HttpContextAccessor.HttpContext!.Request;
+            switch (request.Form["_handler"])
+            {
+                case nameof(ConfirmDeleteAuthenticationConfigurationModel):
+                    ConfirmDeleteAuthenticationConfigurationModel.Purpose = request.Form["ConfirmDeleteAuthenticationConfigurationModel.Purpose"].ToString();
+                    await OnSelectedFormConfirmed();
+                    break;
+            }
+        }
+
+        Configurations = (await Passwordless.GetAuthenticationConfigurationsAsync()).Configurations.Select(GetEntry).ToImmutableArray();
+    }
+
+    public async Task OnSelectedFormConfirmed()
+    {
+        await Passwordless.DeleteAuthenticationConfigurationAsync(new DeleteAuthenticationConfigurationRequest
+        {
+            Purpose = ConfirmDeleteAuthenticationConfigurationModel.Purpose,
+            PerformedBy = HttpContextAccessor.HttpContext!.User.GetName()
+        });
+        
+        NavigationManager.NavigateTo($"app/{CurrentContext.AppId}/settings");
+    }
+
+    public class ConfirmSelectedFormModel
+    {
+        public string Purpose { get; set; } = string.Empty;

```

Comment:
Sanitize the input of the form submission here as well. Or perform form validation. Check out the `Admins` page how to do this.